docs/quickstarts/packet.md

```diff
@@ -0,0 +1,281 @@
+# Lokomotive Packet installation guide
+
+## Contents
+
+* [Introduction](#introduction)
+* [Requirements](#requirements)
+* [Step 1: Install lokoctl](#step-1-install-lokoctl)
+* [Step 2: Set up a temporary working directory](#step-2-set-up-a-temporary-working-directory)
+* [Step 3: Set up credentials from environment variables](#step-3-set-up-credentials-from-environment-variables)
+* [Step 4: Define cluster configuration](#step-4-define-cluster-configuration)
+* [Step 5: Create Lokomotive cluster](#step-5-create-lokomotive-cluster)
+* [Verification](#verification)
+* [Cleanup](#cleanup)
+* [Troubleshooting](#troubleshooting)
+* [Conclusion](#conclusion)
+* [Next steps](#next-steps)
+
+## Introduction
+
+This quickstart guide walks through the steps needed to create a Lokomotive cluster on Packet with
+Flatcar Container Linux using Route53 as the DNS provider.
+
+By the end of this guide, you'll have a working Kubernetes cluster with 3 worker nodes and 3
+controller nodes.
+
+## Requirements
+
+* Basic understanding of Kubernetes concepts.
+* Packet account, Project ID and auth token (sometimes also referred to as [User Level API
+  key](https://www.packet.com/developers/docs/API/getting-started/)).
+* AWS account and IAM credentials (optional for Route53 DNS configuration).
+* AWS Route53 DNS Zone (registered Domain Name or delegated subdomain).
+* Terraform v0.12.x and [terraform-provider-ct](https://github.com/poseidon/terraform-provider-ct)
+  installed locally.
+* Local BGP enabled.
+* An SSH key pair for management access.
+* `kubectl` installed locally to access the Kubernetes cluster.
+
+## Steps
+
+### Step 1: Install lokoctl
+
+Lokoctl is a command-line interface for Lokomotive.
+
+To install, find the [appropriate package](link-to-github-releases-page) for your system
+and download it. Lokoctl is packaged as a `tar.gz` archive.
+
+```console
+wget <URL-TO-DOWNLOAD-LOKOCTL>
+```
+After downloading, untar the package. Lokoctl runs as a single binary named `lokoctl`
+
+```console
+tar xvzf <LOKOCTL_TAR_GZ_ARCHIVE>
+```
+
+The final step is to make sure that the `lokoctl` binary is available on the `PATH`
```

Comment:
This is technically not required, but makes it easier for further referenced commands. Can we maybe mention that somehow?